src/core/control/zoom/ZoomControl.cpp

```diff
@@ -229,7 +229,8 @@ void ZoomControl::initZoomHandler(GtkWidget* window, GtkWidget* widget, XournalV
     gtk_widget_add_events(widget, GDK_TOUCHPAD_GESTURE_MASK);
     g_signal_connect(widget, "scroll-event", G_CALLBACK(onScrolledwindowMainScrollEvent), this);
     g_signal_connect(widget, "event", G_CALLBACK(onTouchpadPinchEvent), this);
-    g_signal_connect(window, "configure-event", G_CALLBACK(onWindowSizeChangedEvent), this);
+    g_signal_connect(window, "signal_size_allocate", G_CALLBACK(onWindowSizeChangedEvent), this);
```

Comment:
The event didn't get triggered when the size-change was initiated by the program, not the user (e.g. when hiding a menu-bar, instead of directly resizing the window). Searching for a solution I stumbled upon  `signal_size_allocate`, which also gets triggered, when these in-program-size-changes happen.
This fixed this problem.

Reply:
I was searching for a solution to detect for widget size changes (as configure-event only reacts, when the whole window size changes). For whatever reason I pushed the wrong signal-event. What actually listens to the widget size change is the size allocate)